spec/unit/message_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'message'
+
+describe Message do
+  describe '.all' do
+    it 'returns message' do
+      Message.create(message: 'This is the a message')
+
+      message = Message.all
```

Comment:
I'd probably pluralise the variable name here since it Message.all returns an array